Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM codeforafrica/hurumap:0.1.1
+FROM hurumap_web:latest
```

Comment:
I'm assuming we're going to change this after cutting a new HURUmap image, right?

Reply:
@kilemensi yes, this will change